spec/requests/api/automate_spec.rb

```diff
@@ -0,0 +1,86 @@
+#
+# REST API Request Tests - /api/automate
+#
+describe "Automate API" do
+  before(:each) do
+    MiqAeDatastore.reset
+    FactoryGirl.create(:miq_ae_domain, :name => "ManageIQ", :tenant_id => @group.tenant.id)
+    FactoryGirl.create(:miq_ae_domain, :name => "Custom",   :tenant_id => @group.tenant.id)
+    system_class = FactoryGirl.create(:miq_ae_class, :name => "System", :namespace => "Custom")
+    FactoryGirl.create(:miq_ae_field, :name    => "on_entry", :class_id => system_class.id,
+                                      :aetype  => "state",    :datatype => "string")
+  end
+
+  context "Automate Queries" do
+    it "returns domains by default" do
+      api_basic_authorize action_identifier(:automate, :read, :collection_actions, :get)
+
+      run_get automate_url
+
+      expect(response).to have_http_status(:ok)
+      expect(response.parsed_body).to a_hash_including(
```

Comment:
Prefer `include` for the top level matcher for readability, `a_hash_including` for  matchers it's composed out of